src/handlers/edit.ts

```diff
@@ -24,8 +24,12 @@ export namespace EditHandler {
       task.description = args.description;
     }
 
-    if (typeof args.by === "string") {
+    if (typeof args.by === "string")
```

Comment:
This `if` doesn't seem to have a clear body

Reply:
Nice catch! I just fixed it